test/Microsoft.DotNet.ToolPackage.Tests/LockFileMatcherTests.cs

```diff
@@ -1,28 +1,82 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
I'm confused by the changes to this file.  Was this file intended to move somewhere given it's no longer a test class?

Reply:
You are right, thanks for catching this. I was only look at the right side didn't realize this is a test file. Something is wrong

Reply:
ah... bad bad merge....

Reply:
that file is fixed